### PR TITLE
feat(governance): add Compliance Standards capability to uipath-governance skill

### PR DESCRIPTION
**Pull request by NishankSiddharth**
https://uipath.atlassian.net/browse/PLT-104173

## What this does

Adds Compliance Standards support to the \`uipath-governance\` skill. Users can check their ISO 42001 posture, apply recommended settings, or remove them — in plain English, no CLI knowledge required.

## User-facing capabilities

| Intent | Example prompt | What happens |
|---|---|---|
| Check posture | "Check my ISO 42001 posture" | Runs coverage, shows Applied/Not Applied per clause with SUMMARY + DETAILS |
| Full apply | "Apply ISO 42001 to my tenant" | Coverage → confirmation → \`state enable tenant\` |
| Scoped apply | "Apply only AI Trust Layer settings from ISO 42001" | Synthesizes formData for matched clauses, collects org-specific values |
| Clause query | "What does clause A.6.2.8 recommend?" | Catalog lookup only, no mutations |
| List configured | "Which compliance standards are active?" | \`state list\` |
| Disable | "Remove ISO 42001 settings from my tenant" | \`state get\` → confirmation → \`state disable\` |
| Org-wide | "Apply ISO 42001 across all tenants" | Lists tenants → coverage per tenant → \`state enable tenant\` per tenant (org-scope enable not implemented on backend) |
| Switch tenant | "Apply this to my staging tenant" | Detects mismatch, shows \`uip login tenant set\` fast path |

The skill never claims a tenant is "compliant" — only that recommended settings are configured. Auditor determines status.

## Files changed

**Skill docs** (\`skills/uipath-governance/\`)
- \`SKILL.md\` — compliance standard routing, tenant-switching, settings terminology, org-wide anti-pattern
- \`references/compliance-pack/\` — 6 plugin impl docs: catalog, coverage, full-apply, partial-apply, disable, query
- \`references/compliance-pack/scripts/\` — synthesize-formdata.md + merge-overrides.md
- \`references/auth-context.md\` — tenant-switching section + mismatch block

**Tests** (\`tests/tasks/uipath-governance/\`)
- 16 smoke/e2e tests + 1 edge case + 1 disambiguation
- \`check_full_apply_report.py\` + \`check_coverage_real_data.py\` — run_command validators
- \`cleanup_compliance_pack.py\` — post_run teardown with robust tenant ID resolution
- \`run-live-e2e.sh\` + \`live-e2e-local.yaml\` — local live-tenant testing

## Key fixes / decisions

| | Detail |
|---|---|
| Org-wide apply | \`state enable organization\` not implemented on backend — iterates over all tenants via \`uip login tenant list\` then \`state enable tenant\` per tenant |
| \`tenantPolicies\` PascalCase | Would have silently wiped all existing tenant policy assignments |
| Session-unique temp dirs | Prevents cross-session contamination |
| Cleanup tenant ID resolution | 3-tier: env vars → \`/.uipath/.auth\` (Docker mount) → \`uip login status\` fallback |
| Verb-gate fix | \`uip gov…\` in anti-pattern text was parsed as stale verb |

## Test plan
- [ ] GH smoke workflow passes on this branch
- [ ] 13+ smoke tests pass
- [ ] \`disambiguation_smoke\` llm_judge score ≥ 0.5

## Coder eval testing

https://github.com/UiPath/skills/actions/runs/27566488239/job/81491733706

image

Generated with Claude Code

### CODE REVIEW

File: tests/tasks/uipath-governance/access-policy/access-policy_lifecycle_e2e.yaml [hunk 1]
```diff
@@ -96,7 +96,3 @@ success_criteria:
     min_count: 3
     weight: 1.0
     pass_threshold: 1.0
-
```
Comment (jianjunwang2) [inline tests/tasks/uipath-governance/access-policy/access-policy_lifecycle_e2e.yaml:98]: Should you revert this part?

File: tests/tasks/uipath-governance/aops-policy/aops-policy_lifecycle_e2e.yaml [hunk 2]
```diff
@@ -86,7 +86,3 @@ success_criteria:
     min_count: 3
     weight: 1.0
     pass_threshold: 1.0
-
```
Comment (jianjunwang2) [inline tests/tasks/uipath-governance/aops-policy/aops-policy_lifecycle_e2e.yaml:88]: Should you revert this part?

File: skills/uipath-governance/references/auth-context.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+UIPATH_TENANT_NAME=$(grep       '^UIPATH_TENANT_NAME='        "$AUTH_FILE" | cut -d'=' -f2-)
+UIPATH_TENANT_ID=$(grep         '^UIPATH_TENANT_ID='          "$AUTH_FILE" | cut -d'=' -f2-)
+UIPATH_ACCESS_TOKEN=$(grep      '^UIPATH_ACCESS_TOKEN='       "$AUTH_FILE" | cut -d'=' -f2-)
+```
+
+## Available fields
+
+| Key | Example | Used by |
+|---|---|---|
+| `UIPATH_URL` | `https://alpha.uipath.com` | principals-lookup (URL construction) |
```
Comment (jianjunwang2) [inline skills/uipath-governance/references/auth-context.md:30]: Regarding org/tenant info, you can also rely OMS cli to get these info, e.g. "uip admin tenants"

File: tests/tasks/uipath-governance/access-policy/access-policy_lifecycle_e2e.yaml (hunk 1 above)
Comment (NishankSiddharth) [reply tests/tasks/uipath-governance/access-policy/access-policy_lifecycle_e2e.yaml:98]: it happened due to incorrect rebasing, reverted this

File: tests/tasks/uipath-governance/aops-policy/aops-policy_lifecycle_e2e.yaml (hunk 2 above)
Comment (NishankSiddharth) [reply tests/tasks/uipath-governance/aops-policy/aops-policy_lifecycle_e2e.yaml:88]: it happened due to incorrect rebasing, reverted this

**Review by jianjunwang2 — APPROVED** [review]

File: skills/uipath-governance/references/auth-context.md (hunk 3 above)
Comment (NishankSiddharth) [reply skills/uipath-governance/references/auth-context.md:30]: sure will keep in mind for future

**Comment by NishankSiddharth** [comment]
Last set of pushes were related to making coder eval test aligned with Coder-eval task quality & prompt-fitness [review](https://uipath-product.slack.com/archives/C0A2T23NJ59/p1781629958542119)

This is the current state of latest coder eval test results, all new compliance tests are passing which contains smoke tests + e2e test against a live tenant.

image


Run1: https://github.com/UiPath/skills/actions/runs/27777306316/job/82192736278
Run2: https://github.com/UiPath/skills/actions/runs/27814379889/job/82311880574
Run3: https://github.com/UiPath/skills/actions/runs/27815413232/job/82315299523

There are some unrelated failing tests related to aops/access policy which are not related to changes in this PR. From what I investigated those failures are mostly due to assertion regex mismatch commands are called correctly, it's mostly tests issue.

**Comment by NishankSiddharth** [comment]
Latest coder eval run for new tests (no failures)
https://github.com/UiPath/skills/actions/runs/28121834773

**Review by t-hsia — APPROVED** [review]

**Review by grant-tannert — APPROVED** [review]

File: skills/uipath-governance/references/compliance-pack/scripts/synthesize-formdata.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+# synthesize-formdata
```
Comment (gabrielavaduva) [inline skills/uipath-governance/references/compliance-pack/scripts/synthesize-formdata.md]: the files in the `scripts` folder should be executable code, the instructions should be in the `references` files and to point to the scripts

File: skills/uipath-governance/references/compliance-pack/scripts/merge-overrides.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
+# merge-overrides
```
Comment (gabrielavaduva) [inline skills/uipath-governance/references/compliance-pack/scripts/merge-overrides.md]: same as below

File: skills/uipath-governance/references/compliance-pack/scripts/synthesize-formdata.md (hunk 4 above)
Comment (NishankSiddharth) [reply skills/uipath-governance/references/compliance-pack/scripts/synthesize-formdata.md]: addressed

File: skills/uipath-governance/references/compliance-pack/scripts/merge-overrides.md (hunk 5 above)
Comment (NishankSiddharth) [reply skills/uipath-governance/references/compliance-pack/scripts/merge-overrides.md]: addressed

**Review by gabrielavaduva — APPROVED** [review]

**Comment by NishankSiddharth** [comment]
Recent commits after last approvals were mostly flaky test fixes
Coder eval latest runs for compliance pack tests: 
1. https://github.com/UiPath/skills/actions/runs/28572928970
2. https://github.com/UiPath/skills/actions/runs/28572941552
